content/en/docs/reference/kubectl/resource_printing/summary.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Summaries"
+linkTitle: "Summaries"
+weight: 1
+type: docs
+description: >
+    Prints Summary of currently working resources and their states
+---
+
+{{< alert color="success" title="TL;DR" >}}
+- Get a Summary of Resources Running in the Cluster
+{{< /alert >}}
+
+# Summarizing Resources
+
+## Motivation
+
+Quickly summarizing a collection of Resources and their state.
```

Comment:
Thanks.
We've realized that a straight conversion isn't right. This proposed tweak should help a lot though:
```suggestion
```